test/sanity/basics/stringPrinter/stringPrinter.wtest

```diff
@@ -0,0 +1,15 @@
+const pepita = object {
+  var energia = 10
+  override method toString() = "pepita"
+}
+
+const printer = new StringPrinter()
```

Comment:
Is the StringPrinter class defined in Wollok?

Reply:
yes, `lib` file (just for the record, already answered this last night)